### ISSUE TITLE
Adding support for the latest iOS 11 Apple Podcast itunes attributes

### ISSUE THREAD

**Issue by razyalov** [rometools/rome#358]
I have created a patch for the changes needed in the iTunes module to support the latest RSS feed spec from Apple as defined in http://podcasts.apple.com/resources/spec/ApplePodcastsSpecUpdatesiOS11.pdf

Attached is the patch of my changes (I don't have permissions to push a pull-request).
[Add_new_Apple_Podcast_spec_support_per_their_latest_update_of_the_iOS_11_and_MacOS_High_Si.patch.zip](https://github.com/rometools/rome/files/1282674/Add_new_Apple_Podcast_spec_support_per_their_latest_update_of_the_iOS_11_and_MacOS_High_Si.patch.zip)

**Comment by mishako** [comment]
Thanks for the patch, this is very needed!

I don't think github has any restrictions on pull requests to public repositories. Did you fork the repository first? Try forking it, pushing to your fork and then creating a PR.
https://help.github.com/categories/collaborating-with-issues-and-pull-requests

**Comment by razyalov** [comment]
will do.

Raz

**Comment by razyalov** [comment]
I did it.
For some reason the build failed on travis-ci although when I ran mvn verify with idk 7 on my computer if passed with no issues:

Tests run: 11, Failures: 0, Errors: 0, Skipped: 3

[INFO] 
[INFO] --- maven-jar-plugin:3.0.2:jar (default-jar) @ rome-propono ---
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] rome-parent ........................................ SUCCESS [  0.002 s]
[INFO] rome-utils ......................................... SUCCESS [  1.637 s]
[INFO] rome ............................................... SUCCESS [  6.165 s]
[INFO] rome-fetcher ....................................... SUCCESS [  1.326 s]
[INFO] rome-certiorem ..................................... SUCCESS [  1.518 s]
[INFO] rome-certiorem-webapp .............................. SUCCESS [  0.516 s]
[INFO] rome-modules ....................................... SUCCESS [  4.762 s]
[INFO] rome-opml .......................................... SUCCESS [  1.398 s]
[INFO] rome-osgi-test ..................................... SUCCESS [  5.618 s]
[INFO] rome-propono ....................................... SUCCESS [  4.151 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 27.294 s
[INFO] Finished at: 2017-09-08T10:33:48-07:00
[INFO] Final Memory: 20M/244M
[INFO] ------------------------------------------------------------------------


I wonder if this is a travis-ci configuration issue with the idk version.

Raz

I

**Comment by mishako** [comment]
Yes, it's a travis-ci configuration issue: https://github.com/travis-ci/travis-ci/issues/7884
I changed it to openjdk7: 6b4dd5b

**Comment by razyalov** [comment]
great. Let me know if you need any more help on my side.

I’m already using a local build of this for now, until you guys release the new version and it works for generating the RSS feed with no issues.


Raz

**Comment by mishako** [comment]
@razyalov I released a new version (1.8.0). Now @PatrickGotthard has to promote it to Maven Central.

**Comment by PatrickGotthard** [comment]
Version 1.8.0 is released. It will be available on Maven Central in a few hours 👍

**Comment by PatrickGotthard** [comment]
@mishako I have adjusted the parent POM according to http://central.sonatype.org/pages/apache-maven.html - hopefully future versions will be released automatically

**Comment by mihxil** [comment]
Could it be that the resulting feeds are invalid? 

https://validator.w3.org/feed/check.cgi?url=https://rs.vpro.nl/v3/api/feeds/podcast/POMS_S_VPRO_11660029

Does someone know whether apple provides an updated dtd too?

**Comment by mishako** [comment]
@mihxil I wouldn't expect w3c validator to support latest specifications.
Don't know about apple's dtd.

**Comment by mihxil** [comment]
I wouldn't either. But it seems that apple is rejecting podcasts for inclusion in itunes because 'they don't validate'.

**Comment by razyalov** [comment]
Usually Apple will fail validation for content, not DTD validation. 

First check your feed here: http://castfeedvalidator.com/
Also make sure your artwork is correct both from a dimensions perspective (1200x1200 - 1600x1600 pixels) and size (up to 2MB). 
These numbers are from the top of my head so double check them on the podcast spec page by Apple.
